### ISSUE TITLE
Live-Updates doesn't work on Raspberry Pi 3[BUG]

### ISSUE THREAD

**Issue by tpt5cu** [plotly/dash#962]
-  replace the result of `pip list | grep dash` below
```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
dash                               1.3.0      
dash-core-components               1.2.0      
dash-daq                           0.2.1      
dash-html-components               1.0.1      
dash-renderer                      1.1.0      
dash-table                         4.3.0
```
-  if frontend related, tell us your Browser, Version and OS

Raspbian GNU/Linux 9 using Chromium

**Describe the bug**

When following the Live-Updates  tutorial from https://dash.plot.ly/live-updates, the graphs appear in the web browser, but the graphs are empty. Below are screen shots of what happened vs. what is expected respectively. 

<img width="1438" alt="Screen Shot 2019-10-08 at 4 45 22 PM" src="https://user-images.githubusercontent.com/22273408/66522439-6f622e00-eabb-11e9-86b6-40f6284fcea5.png">

<img width="1431" alt="Screen Shot 2019-10-08 at 4 46 23 PM" src="https://user-images.githubusercontent.com/22273408/66522446-74bf7880-eabb-11e9-9e8c-9d8259838f6a.png">

SOLUTION:

The solution was to simply increase the interval to 1 * 10000 ms. Then the live-updates worked as expected. Not entirely sure why, but wanted to bring this issue up. Thanks!

**Comment by alexcjohnson** [comment]
most likely a duplicate of #943 - @tpt5cu can you see if this is the case by looking at the requests in the network tab to see if a new one fires before the previous responds?

**Comment by tpt5cu** [comment]
@alexcjohnson after taking a look at the network tab it looks like the _dash-update-component request keeps firing. There is a json response of the text I want in the html span. However no _reload-hash request is ever fired (or at least does so only once to load the chart, but not populate it with data). _dash-update-component keeps firing. When I increase the interval, _reload-hash fires in equal proportions to _dash-update-component. However each request appears to have some sort of response in the response tab in chromium. Is this helpful? Does this make sense? I am not super well versed in web development, apologies in advance.
